src/tests/vos_perf.c

```diff
@@ -949,6 +961,9 @@ main(int argc, char **argv)
 	stride_buf_fini();
 	dts_ctx_fini(&ts_ctx);
 
+#ifdef ULT_MMAP_STACK
+	stack_pool_destroy(sp);
```

Comment:
similar to prior comment, if stack_pool_destroy is defined as (void)0 when ULT_MMAP_STACK is undefined, there is no need for inline ifdef

Reply:
> similar to prior comment, if stack_pool_destroy is defined as (void)0 when ULT_MMAP_STACK is undefined, there is no need for inline ifdef

hmmm, did I finally and correctly understand that you want me to "#define stack_pool_[create,destroy] (void)0" ifndef ULT_MMAP_STACK in stack_mmap.h ?

Reply:
yeah, it's a minor nit, just would make the code a bit cleaner.